src/commands/protocols/publish/receiver/v1.0.0/v1-0-0-handle-store-init-command.js

```diff
@@ -146,7 +146,7 @@ class HandleStoreInitCommand extends HandleProtocolMessageCommand {
 
             const ask = this.blockchainModuleManager.convertToWei(blockchain, peerRecord.ask);
 
-            return BigNumber.from(ask);
+            return this.blockchainModuleManager.toBigNumber(blockchain, ask);
```

Comment:
I think we should move all unit and number conversion somewhere else, since it is not really dependent on the blockchain implementation